account_check_printing_report_dlt103/__manifest__.py

```diff
@@ -16,7 +15,6 @@
     "data": [
         "data/report_paperformat.xml",
         "data/report_paperformat_parameter.xml",
-        "data/account_payment_check_report_data.xml",
```

Comment:
Why did you delete this?

Reply:
That file is used to add option dlt103 for check printing, which is handled with selection field account_check_printing_layout on payment.journal in v15, so no longer a need for that file with that declaration way